python_modules/libraries/dagster-aws/dagster_aws/ecs/launcher.py

```diff
@@ -327,5 +321,26 @@ def _task_definition(self, family, metadata, image, container_context):
             include_sidecars=self.include_sidecars,
         )
 
+    def _reuse_task_definition(self, task_definition, metadata, image, secrets):
```

Comment:
I had to go back and find our conversation when this was first introduced: https://github.com/dagster-io/dagster/pull/7009#discussion_r825213675

I still have this lingering feeling that this is going to keep missing stuff (but I have no concrete suggestions other than the one we discussed there, of writing our own version of the mapping from input task def to output task def, which I agree feels very brittle)

Reply:
I agree it will but at least this gives us a single spot to capture that logic as it emerges.